packages/access-api/test/access-authorize.test.js

```diff
@@ -149,31 +162,28 @@ describe('access/authorize', function () {
     )
     assert.deepEqual(
       claimedDelegations.length,
-      1,
+      2,
       'should have claimed delegation(s)'
     )
 
     const claimedDelegationIssuedByService = claimedDelegations.find((d) => {
-      if (!('cid' in d.proofs[0])) {
+      if (!('cid' in d)) {
         throw new Error('proof must be delegation')
       }
-      return d.proofs[0].issuer.did() === service.did()
+      return d.issuer.did() === service.did()
     })
+
     assert.ok(
       claimedDelegationIssuedByService,
       'should claim ucan/attest with proof.iss=service'
     )
 
-    // we can use claimedDelegationIssuedByService to invoke access/claim as iss=accountDID
-    const account = issuer.withDID(accountDID)
+    // we can use delegations to invoke access/claim with=accountDID
     const claimAsAccount = Access.claim.invoke({
-      issuer: account,
+      issuer: agent,
```

Comment:
We should be able to issue this with `account` right?

Reply:
after discussing with @Gozala I realized this isn't important. `with` is still `accountDID` so it still selects delegations with aud=accountDID. That's what @Gozala meant i think.